src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -72,7 +72,7 @@ public final class InlineConfigParser {
 
     /** A pattern to find the string: "// violation, explanation". */
     private static final Pattern VIOLATION_WITH_EXPLANATION_PATTERN = Pattern
-            .compile(".*//\\s*violation,\\s.+\\s(?:'(.*)')?$");
+            .compile(".*//\\s*violation,\\s+(?:'(.*)')?$");
```

Comment:
I have no idea how we came up with this weird regexp, but it did not even match it's own javadoc. I have amended it because we have a lot of violation messages like this in examples.